portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,159 @@
-}
+@import url('https://fonts.googleapis.com/css?family=Raleway');
+
+  #content {
+    margin-left: auto;
+    margin-right: auto;
+    width: 700px;
+    transition: opacity 1s ease-in;
+    transition-delay: 1s;
+    position: relative;
+    bottom: 25px;
+  }
+
+  p {
+    letter-spacing: 1px;
+  }
+
+  h1, h2, h3, h4, h5, h6 {
+    color: rgb(241, 129, 0);
+  }
+  
+  body {
+    color: rgba(255, 232, 202, 0.733);
+    margin: 0;
+    background-color: rgb(31, 31, 31);
+    font-family: 'Raleway', sans-serif;
+  }
+
+  #intro-wrapper {
+    display: flex;
+    justify-content: center;
+    opacity: 1;
+    transition: opacity 1s ease-out;
+  }
+
+  #intro {
+    font-size: 1.5rem;
+    border-radius: 20px;
+    padding: 35px;
+    position: relative;
+    top: 250px;
+    font-weight: 100;
+    letter-spacing: 2px;
+    animation: border-flicker 2.5s linear infinite;
+  }
+
+  @keyframes border-flicker {
+    0% {
+      border: 5px solid rgba(241, 129, 0, 0.75);
+      box-shadow: 0px 0px 80px 15px rgba(241, 129, 0, 0.25);
+    }
+    50% {
+      border: 5px solid rgba(241, 129, 0, 0.3);
+      box-shadow: 0px 0px 80px 15px rgb(241, 129, 0);
+    }
+    100% {
+      border: 5px solid rgba(241, 129, 0, 0.75);
+      box-shadow: 0px 0px 80px 15px rgba(241, 129, 0, 0.25);
+    }
+  }
+
+  #name {
+    font-size: 2.5em;
+    font-weight: 550;
+    letter-spacing: 15px;
+    padding-top: 10px;
+    display: block;
+    color: rgb(241, 129, 0);
+  }
+
+  #intro-wrapper.hide, #content.hide{
+    opacity: 0;
+  }
+
+  .logo {
+    width: 60px;
+    height: 60px;
+    background-size: 100%;
+    background-repeat: no-repeat;
+    background-position: center;
+    border-radius: 30px;
+    background-color: rgb(59, 59, 59);
+    display: inline-block;
+    margin-right: 5px;
+  }
+
+  .logo:hover {
+    background-color: rgb(241, 129, 0);
+  }
+
+  #linkedin.logo {
+    background-image: url("images/linkedin.png");
+  }
+
+  #github.logo {
+    background-image: url("images/github.png");
+  }
+
+  #instagram.logo {
+    background-image: url("images/insta.png");
+    background-size: 65%;
+  }
+
+  #next.logo {
+    border: 0;
+    position: absolute;
+    right: 0;
+  }
+
+  #dp {
+    width: 150px;
+    height: 150px;
+    background-size: 100%;
+    border-radius: 75px;
+    display: inline-block;
+    background-image: url("images/dp.jpg");
+    margin-left: 20px;
+  }
+
+  #dp-click {
+    font-weight: 800;
+  }
+
+  #dp-click:hover {
+    color: white;
+  }
+
+  .section {
+    padding-bottom: 13vh;
+  }
+
+  button {
+    background-color: rgb(59, 59, 59);
+    color: rgba(255, 232, 202, 0.733);
+    font-family: 'Raleway', sans-serif;
+    font-size: 1rem;
+    border-radius: 5px;
+    padding: 5px;
+  }
+
+  .wp-img {
+    height: 140px;
+    transition: height 0.7s;
+  }
+
+  .wp-img.hide {
+    height: 0;
+  }
+
+  b {
+    color: whitesmoke;
+  }
+
+  .travel-img {
+    height: 149px;
+  }
+
+  .sea-img {
+    height: 198px;
+  }
```

Comment:
it is a good convention to add endline at the end of the file

Reply:
Noted! Will practice this.